Articles/Blog/2024-07-BSSwFellows2023.md

```diff
@@ -0,0 +1,208 @@
+
+</div>
+</div>
+
+*Perspectives on the BSSw Fellowship Program:* Before returning to grad school, I worked as a research software engineer at Purdue, developing Jupyter-based web applications to enhance the accessibility and reproducibility of research code and data. The BSSw fellowship provided me with rare and valuable time to reflect on and disseminate lessons learned from my experiences. More unexpectedly, it gave me the space to reflect and pivot my dissertation topic to align closely with my passion for scientific notebooks. Jupyter notebooks are incredibly flexible tools, which makes them amenable to innovative uses but also challenging to characterize. For my dissertation, I aim to explore the boundaries and scope of Jupyter Notebooks in scientific contexts, with a particular focus on their findability and reproducibility. These research goals will in turn serve Jupyter4Science site, which I envision as a hub for a well-scoped community of practice. To further the goal of community-building, I have published a page to the site about the various ways others can [contribute](https://jupyter4.science/contribute/) to the project.
+
+*Advice for new (prospective) BSSw Fellows:* For my proposal, I decided to use my funding in part to hire students to assist me with the production of my fellowship deliverables. I had mentored undergraduate students before, but only from within the confines of a structured program with predefined goals. This time, I was stepping outside the boundaries of traditional internships with two full-time undergraduates. Hiring and onboarding students during the summer required a massive upfront effort as I developed the internship program on the fly. It took a few weeks for us to hit our stride, but over the summer, I watched them grow in their capacity for project management and intellectual independence at an astounding rate. They stayed on with me well past the initial time frame and evolved into integral team members, making valuable contributions to the Jupyter4Science site and winning a Best Student Poster award for their submission to the inaugural US-RSE conference in October 2023. I would recommend involving students, and especially undergraduates, to anyone willing to prioritize their students' growth over immediate project outcomes. In my experience, involving students "save me time," however, their fresh perspectives drastically enriched the project.
```

Comment:
The text says 'involving students "save me time,"', which doesn't make sense.  I suspect that there's a 'don't' missing, or something like that?  Whatever the right answer is, please fix.